controllers/rest/global_initializers.go

```diff
@@ -10,17 +11,29 @@ import (
 func Security(ctxt context.Context, api *RESTAPI, swagger *openapi3.Swagger) (context.Context, error) {
 	middlewares := GetOperationMiddlewares(ctxt)
 	found := false
-	for _, scheme := range swagger.Components.SecuritySchemes {
+	for key, scheme := range swagger.Components.SecuritySchemes {
```

Comment:
Loop through the global security and confirm that they exist in the security schemes (right now it's doing the opposite)